pkg/config/config.go

```diff
@@ -38,8 +39,8 @@ func RenderK0sConfig(proxyRegistryDomain string) *k0sconfig.ClusterConfig {
 	return cfg
 }
 
-// extractK0sConfigPatch extracts the k0s config portion of the provided patch.
-func extractK0sConfigPatch(raw string) (string, error) {
+// ExtractK0sConfigPatch extracts the k0s config portion of the provided patch.
+func ExtractK0sConfigPatch(raw string) (string, error) {
```

Comment:
Why was this promoted to an external function? I don't see it called anywhere new